README.rst

```diff
@@ -121,6 +121,7 @@ Scrapers available for:
 - `https://www.bodybuilding.com/ <https://www.bodybuilding.com/>`_
 - `https://bonappetit.com/ <https://bonappetit.com>`_
 - `https://bongeats.com/ <https://bongeats.com/>`_
+- `https://books.ottolenghi.co.uk <https://books.ottolenghi.co.uk/>`_
```

Comment:
```suggestion
- `https://books.ottolenghi.co.uk <https://books.ottolenghi.co.uk/>`_ (*)
```

Nitpick: let's add an asterisk annotation to this scraper to note that it runs on non-public HTML content (a few other scrapers have this annotation for similar reasons).